utils/utils.go

```diff
@@ -51,6 +53,74 @@ func GenerateNoteID() string {
 	return string(result)
 }
 
+// This function generate note names for unnamed notes.
+// The note name is as such : (book)_note_(current number of note + 1)
+// E.g, foo_note_001, foo_note_002, foo_note_003
+// The name consists of 3 parts :
+// 1. book is the name of the book.
+// 2. _note_ is a constant substring.
+// 3. the number of note is the current number of note + 1.
+func GenerateNoteName() (string, error) {
+	var result string
+	constant_middle := "_note_"
+	result = ""
+
+	book, err := GetCurrentBook()
+	if err != nil {
+		return result, err
+	}
+
+	json_data, err := GetDnote()
+	if err != nil {
+		return result, err
+	}
+
+	var note_names []string
+	for _, note := range json_data[book] {
+		if strings.Contains(note.Name, constant_middle) {
+			note_names = append(note_names, note.Name)
+		}
+	}
+
+	var note_num_slice []int
+	if note_names != nil {
+		for _, note_name := range note_names {
```

Comment:
Why would you do that? If it were to be implemented, it wouldn't just take one arguments, it would be 3, and it would only work for a certain format of note names.